### PR TITLE
Added Block Page

### PR DESCRIPTION
**Pull request by Federicofilini**

### CODE REVIEW

File: blog.css [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+
+.blog_section {
+  margin-inline: auto;
+  max-width: 1740px;
+}
+.header-nav_block {
+  position: relative;
+  background-color: #1d1c28;
+  min-height: 375px;
+  padding: 2.5%;
```
Comment (andacenko) [inline blog.css:21]: Use padding in px.

File: blog.css [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+  z-index: 1;
+}
+.header-nav_block::after {
+  content: "";
+  background-image: url(./img/Backgrounds/2fa987cb1a683f967534cdbd93011254.webp);
+  position: absolute;
+  background-position: center;
+  background-size: cover;
+  width: 100%;
+  height: 375px;
```
Comment (andacenko) [inline blog.css:35]: height should be flexible. Use height: 100% (or remove height and use top:0)

File: blog.css [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+  content: "";
+  background-image: url(./img/Backgrounds/Rectangle\ 778.svg);
+  position: absolute;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  width: 100%;
+  height: 177px;
+  position: absolute;
+  bottom: 0;
```
Comment (andacenko) [inline blog.css:52]: Please add left:0;

File: blog.css [hunk 4]
```diff
@@ -0,0 +1,350 @@
...
+  z-index: 1;
+}
+
+.blog_header {
+  font-size: 52px;
+  font-weight: 400;
+  letter-spacing: -0.5px;
+  font-family: Bakbak One;
```
Comment (andacenko) [inline blog.css:60]: Please use list of font famalies. font-family: Bakbak One, Times New Roman, Georgia, Serif;

File: blog.css [hunk 5]
```diff
@@ -0,0 +1,350 @@
...
+  background-size: cover;
+  width: 100%;
+  height: 177px;
+  position: absolute;
+  bottom: 0;
+  z-index: 1;
+}
+
+.blog_header {
+  font-size: 52px;
```
Comment (andacenko) [inline blog.css:57]: Add line-height

File: blog.css [hunk 6]
```diff
@@ -0,0 +1,350 @@
...
+}
+
+.blog-page_breadcrumbs {
+  position: relative;
+  display: flex;
+  align-items: center;
+  font-size: 14px;
+  font-weight: 600;
+}
+.blog-page_breadcrumbs:last-child {
```
Comment (andacenko) [inline blog.css:71]: It's style for .blog-page_breadcrumbs li.
Not for last-child.
We set styles for links and plain text.

File: blog.css [hunk 7]
```diff
@@ -0,0 +1,350 @@
...
+  letter-spacing: -0.5px;
+  font-family: Bakbak One;
+  margin-bottom: 20px;
+}
+
+.blog-page_breadcrumbs {
+  position: relative;
+  display: flex;
+  align-items: center;
+  font-size: 14px;
```
Comment (andacenko) [inline blog.css:68]: Add line-height

File: blog.css [hunk 8]
```diff
@@ -0,0 +1,350 @@
...
+.blog-page_link {
+  color: #f8f7ff;
+  text-decoration: none;
+  margin-right: 24px;
+}
+.blog-page_link:hover {
+  border-bottom: 2px solid #0265dc;
+}
+
+.blog-page_breadcrumbs::before {
```
Comment (andacenko) [inline blog.css:83]: It's should be :before element for li
`.blog-page_breadcrumbs li::before {}`
and remove :before for first li
```
.blog-page_breadcrumbs li:first-shild::before{
display: none
}
```

File: blog.css [hunk 9]
```diff
@@ -0,0 +1,350 @@
...
+.blog-page_breadcrumbs::after {
+  content: "";
+  position: absolute;
+  top: 6px;
+  left: 140px;
+  width: 2px;
+  height: 16px;
+  background: #d0d7fb2b;
+  border-radius: 2px;
+}
```
Comment (andacenko) [inline blog.css:102]: Please delete block .blog-page_breadcrumbs::after

File: blog.css [hunk 10]
```diff
@@ -0,0 +1,350 @@
...
+  font-size: 14px;
+  font-weight: 600;
+}
+.blog-page_breadcrumbs:last-child {
+  color: #e7eef7a8;
+}
+.blog-page_link {
+  color: #f8f7ff;
+  text-decoration: none;
+  margin-right: 24px;
```
Comment (andacenko) [inline blog.css:77]: Delete margin here

File: blog.css [hunk 11]
```diff
@@ -0,0 +1,350 @@
...
+  font-size: 52px;
+  font-weight: 400;
+  letter-spacing: -0.5px;
+  font-family: Bakbak One;
+  margin-bottom: 20px;
+}
+
+.blog-page_breadcrumbs {
+  position: relative;
+  display: flex;
```
Comment (andacenko) [inline blog.css:66]: Add gap: 24px

File: blog.css [hunk 12]
```diff
@@ -0,0 +1,350 @@
...
+  left: 140px;
+  width: 2px;
+  height: 16px;
+  background: #d0d7fb2b;
+  border-radius: 2px;
+}
+/* --------------------------------------------Main-- */
+.ourblog-info__block {
+  margin-inline: auto;
+  max-width: 1728px;
```
Comment (andacenko) [inline blog.css:106]: Delete styles
  ```
margin-inline: auto;
  max-width: 1728px; 
```
we have them in the index.css

File: blog.css [hunk 13]
```diff
@@ -0,0 +1,350 @@
...
+  display: flex;
+  justify-content: center;
+  padding: 64px;
```
Comment (andacenko) [inline blog.css:109]: Set left and right padding 15px

File: blog.css (hunk 1 above)
Comment (andacenko) [reply blog.css:21]: Set left and right padding 15px

File: blog.css [hunk 14]
```diff
@@ -0,0 +1,350 @@
...
+  gap: 32px;
+}
+.aside {
+  font-size: 14px;
+  font-weight: 600;
+  flex-basis: 328px;
+  flex-grow: 0;
+  flex-shrink: 0;
```
Comment (andacenko) [inline blog.css:117]: Add gap 21px

File: blog.css [hunk 15]
```diff
@@ -0,0 +1,350 @@
...
+}
+.side-bar_block {
+  background-color: #1d1c28;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  padding: 24px;
+  margin-bottom: 21px;
```
Comment (andacenko) [inline blog.css:125]: Please remove 
  ```
position: relative;
  margin-bottom: 21px;
```

File: blog.css [hunk 16]
```diff
@@ -0,0 +1,350 @@
...
+  justify-content: center;
+
+  color: #f8f7ff;
+  border: 2px solid rgba(208, 215, 251, 0.17);
+  padding: 6px 12px;
+}
+
+.side-bar_header {
+  color: #f8f7ff;
+  font-size: 24px;
```
Comment (andacenko) [inline blog.css:150]: Add line-height

File: blog.css [hunk 17]
```diff
@@ -0,0 +1,350 @@
...
+  font-weight: 400;
+  font-family: Bakbak One;
```
Comment (andacenko) [inline blog.css:152]: Add full font-family

File: blog.css [hunk 18]
```diff
@@ -0,0 +1,350 @@
...
+.list-two {
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 8px;
+}
+.list-two a {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
```
Comment (andacenko) [inline blog.css:142]: Delete 
  ```
display: flex;
  align-items: center;
  justify-content: center;
```
Add 
`display: block;`

File: blog.css [hunk 19]
```diff
@@ -0,0 +1,350 @@
...
+}
+
+.side-bar_link {
+  color: #f8f7ff;
+  font-weight: 600;
+}
+
+/* --------------------------------------------- */
+.ourblog-details__block {
+  max-width: 1024px;
```
Comment (andacenko) [inline blog.css:165]: Delete max-width: 1024px; here

File: blog.css [hunk 20]
```diff
@@ -0,0 +1,350 @@
...
+  font-size: 28px;
+  font-weight: 400;
+  font-family: Bakbak One;
+}
+
+.ourblog-details__block figcaption {
+  left: 21px;
+  bottom: 12px;
+  position: absolute;
+  color: #fff;
```
Comment (andacenko) [inline blog.css:201]: Please add correct font-size, font-weight, line-height here

File: blog.css [hunk 21]
```diff
@@ -0,0 +1,350 @@
...
+  gap: 32px;
+}
+.content_text_link {
+  color: #78bbfa;
+}
+.ourblog-details__block h2 {
+  color: #f8f7ff;
+  font-size: 36px;
+  font-weight: 400;
+  font-family: Bakbak One;
```
Comment (andacenko) [inline blog.css:177]: Add full f-family and line-height

File: blog.css [hunk 22]
```diff
@@ -0,0 +1,350 @@
...
+.ourblog-details__block figure {
+  position: relative;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.gaming-button {
+ 
+  width: min-content;
+  gap: 8px;
```
Comment (andacenko) [inline blog.css:212]: Delete gap, we don't have flex here

File: blog.css [hunk 23]
```diff
@@ -0,0 +1,350 @@
...
+  color: #f8f7ff;
+  
+  font-size: 18px;
+  font-weight: 600;
+  background-color: #0265dc;
```
Comment (andacenko) [inline blog.css:217]: We can delete bg-color. We have it on class .is-blue and conect it from index.css

File: blog.css [hunk 24]
```diff
@@ -0,0 +1,350 @@
...
+  font-weight: 400;
+  font-family: Bakbak One;
+}
+.ourblog-details__block p {
+  color: #f8f7ff;
+  font-size: 18px;
+  font-weight: 400;
+  font-family: Poppins;
+  line-height: 28px;
+}
```
Comment (andacenko) [inline blog.css:185]: Delete whole block. We have this styles on the body as default.

File: blog.css [hunk 25]
```diff
@@ -0,0 +1,350 @@
...
+
+/* --------------------------------------------- */
+.ourblog-details__block {
+  max-width: 1024px;
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+}
+.content_text_link {
+  color: #78bbfa;
```
Comment (andacenko) [inline blog.css:171]: It should be color for every link in the project (default style for `<a>`)

File: blog.css [hunk 26]
```diff
@@ -0,0 +1,350 @@
...
+  color: #f8f7ff;
+  
+  font-size: 18px;
+  font-weight: 600;
+  background-color: #0265dc;
+  padding: 10px 24px;
+}
+
+.blog-content_list {
+  list-style: none;
```
Comment (andacenko) [inline blog.css:222]: `list-style: disc;`

File: blog.css [hunk 27]
```diff
@@ -0,0 +1,350 @@
...
+  padding-left: 10px;
+  position: relative;
```
Comment (andacenko) [inline blog.css:224]: Delete pos:relative

File: blog.css [hunk 28]
```diff
@@ -0,0 +1,350 @@
...
+}
+.content_item::before {
+  content: "•";
+  position: absolute;
+  left: -15px;
+  color: #f8f7ff;
+  font-size: 20px;
+  line-height: 1.5;
+  font-family: Arial, Helvetica, sans-serif;
+}
```
Comment (andacenko) [inline blog.css:234]: Please delete this bloc

File: blog.html [hunk 29]
```diff
@@ -0,0 +1,478 @@
...
+            Ethereum’s blockchain with real-world value. Players use the
+            <a href="#" class="content_text_link" >Basilisks they have</a>
+            gathered to battle other players to win ether (ETH). In Balthazar
+            Dragons, players are immersed in a 3D open world to explore and
+            capture dragon-like beasts called Basilisks. Balthazar Dragons is an
+            upcoming fantasy role-playing game developed on the Ethereum
+            blockchain by a decentralized autonomous organization (DAO) called
+            the Balthazar DAO.
+          </p>
+          <ol class="blog-content_list">
```
Comment (andacenko) [inline blog.html:195]: Change on the `<ul>`

File: blog.html [hunk 30]
```diff
@@ -0,0 +1,478 @@
...
+                <a href="#" class="content_text_link">game developed</a> on the Ethereum blockchain by a
+                decentralized autonomous organization (DAO) called the Balthazar
+                DAO.
+              </p>
+            </li>
+          </ol>
+
+          <ul class="blog-label__container">
+            <li class="blog-label__list">
+              <p class="blog-logo__text">List name 1</p>
```
Comment (andacenko) [inline blog.html:218]: We shouldn't use `<p>` here. Just `<li>text</li>`

File: blog.css [hunk 31]
```diff
@@ -0,0 +1,350 @@
...
+.blog-logo__text{
+  margin-bottom: 8px;
```
Comment (andacenko) [inline blog.css:236]: It style for .blog-label__container li

File: blog.css [hunk 32]
```diff
@@ -0,0 +1,350 @@
...
+}
+
+.blog-logo__text::before {
```
Comment (andacenko) [inline blog.css:239]: Change selector on .blog-label__container li::before

File: blog.css [hunk 33]
```diff
@@ -0,0 +1,350 @@
...
+  vertical-align: middle;
+}
+/* ---------------------------------------Blockquote */
+
+.blog__blockquote {
+  max-width: max-content;
+  background-color: #002754;
+  border-radius: 10px;
+  padding: 16px;
+  gap: 8px;
```
Comment (andacenko) [inline blog.css:256]: Delete wax-width and gap

File: blog.css [hunk 34]
```diff
@@ -0,0 +1,350 @@
...
+}
+.blog__blockquote--text {
+  color: #f8f7ff;
+
+  font-size: 24px;
+  font-weight: 600;
+  font-family: Poppins;
```
Comment (andacenko) [inline blog.css:263]: Remove all.
Add 
 ```
 font-size: 24px;
  line-height: 30px;
  font-weight: 600;
```

File: blog.css [hunk 35]
```diff
@@ -0,0 +1,350 @@
...
+  flex-wrap: wrap;
+  gap: 2%;
+}
+.foto_box figure {
+  position: relative;
+}
+
+.foto-two {
+  max-width: 100%;
+  height: auto;
```
Comment (andacenko) [inline blog.css:281]: It should be default styles for all images
Add somewhere near body
```
img {
  max-width: 100%;
  height: auto;
  vertical-align: top;
}
```

File: blog.css [hunk 36]
```diff
@@ -0,0 +1,350 @@
...
+  color: #f5f8fb;
+}
+
+.socials__box,
+.tags_box {
+  display: flex;
+
+  align-items: center;
+  gap: 16px;
+}
```
Comment (andacenko) [inline blog.css:306]: Add font styles for title:
```
 .socials__box span,
.tags_box span {...}
```

File: blog.css [hunk 37]
```diff
@@ -0,0 +1,350 @@
...
+  color: #fff;
+}
+/* --------------------------------------- */
+.ourblog-details_footer {
+  display: flex;
+  justify-content: space-between;
+  gap: 12px;
+}
+.ourblog-details_footer{
+  color: #f5f8fb;
```
Comment (andacenko) [inline blog.css:297]: Move color: #f5f8fb; to the line 294. Use one styles block.

File: blog.css [hunk 38]
```diff
@@ -0,0 +1,350 @@
...
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+}
+.tag-link,
+.socials_link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
```
Comment (andacenko) [inline blog.css:318]: Remove
```
display: flex;
  align-items: center;
  justify-content: center;
  gap: 8px;
```
Add
`display: block;`

File: blog.html [hunk 39]
```diff
@@ -0,0 +1,478 @@
...
+                </li>
+                <li class="tag_item">
+                  <a href="" class="tag-link is-white">Crypto</a>
+                </li>
+              </ul>
+            </div>
+            <div class="socials__box">
+              <span>Share:</span>
+              <ul class="socials_list">
+                <li class="socials_item">
```
Comment (andacenko) [inline blog.html:320]: Delete class="socials_item"

File: blog.html [hunk 40]
```diff
@@ -0,0 +1,478 @@
...
+                  <a href="" class="socials_link is-white"
```
Comment (andacenko) [inline blog.html:321]: Delete additional class 'is-white'

File: blog.html [hunk 41]
```diff
@@ -0,0 +1,478 @@
...
+          </footer>
+        </article>
+      </main>
+    
+    <!-- ------------------------------------------------------------------------Footer -->
+    <footer class="footer">
+      <div class="footer-countainer">
+        <div class="footer_bar_box">
+          <div class="footer_logo_box">
+            <strong class="logo-footer"
```
Comment (andacenko) [inline blog.html:360]: Footer logo should be a link `<a href="index.html></a>`

File: blog.html [hunk 42]
```diff
@@ -0,0 +1,478 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Blog Page</title>
+    <link rel="stylesheet" href="./index.css" />
+  </head>
+  <body>
+    <!-- --------------------------------------------------------------------------Header -->
+    <header class="header">
+      <h1 class="logo">
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply blog.html:12]: Done

File: blog.html [hunk 43]
```diff
@@ -0,0 +1,478 @@
...
+        <img
+          src="./img/Logo/logo.svg"
+          alt="Logo"
+          width="36"
+          height="40"
+        />BINABOX
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply blog.html:18]: Done

File: blog.css (hunk 1 above)
Comment (Federicofilini) [reply blog.css:21]: Done

File: blog.css (hunk 2 above)
Comment (Federicofilini) [reply blog.css:35]: Done

File: blog.css (hunk 3 above)
Comment (Federicofilini) [reply blog.css:52]: Done

File: blog.css (hunk 4 above)
Comment (Federicofilini) [reply blog.css:60]: Done

File: blog.css (hunk 5 above)
Comment (Federicofilini) [reply blog.css:57]: Done

File: blog.css (hunk 6 above)
Comment (Federicofilini) [reply blog.css:71]: Done

File: blog.css (hunk 7 above)
Comment (Federicofilini) [reply blog.css:68]: Done

File: blog.html [hunk 44]
```diff
@@ -0,0 +1,478 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
[the remark this answers is not in the document]
Comment (Federicofilini) [reply blog.html:5]: Done

File: blog.css (hunk 8 above)
Comment (Federicofilini) [reply blog.css:83]: Done

File: blog.css (hunk 9 above)
Comment (Federicofilini) [reply blog.css:102]: Done

File: blog.css (hunk 11 above)
Comment (Federicofilini) [reply blog.css:66]: Done

File: blog.css (hunk 12 above)
Comment (Federicofilini) [reply blog.css:106]: Done

File: blog.css (hunk 13 above)
Comment (Federicofilini) [reply blog.css:109]: Done

File: blog.html (hunk 41 above)
Comment (Federicofilini) [reply blog.html:360]: Done

File: blog.css (hunk 14 above)
Comment (Federicofilini) [reply blog.css:117]: Done

File: blog.css (hunk 15 above)
Comment (Federicofilini) [reply blog.css:125]: Done

File: blog.css (hunk 16 above)
Comment (Federicofilini) [reply blog.css:150]: Done

File: blog.css (hunk 17 above)
Comment (Federicofilini) [reply blog.css:152]: Done

File: blog.css (hunk 18 above)
Comment (Federicofilini) [reply blog.css:142]: Done

File: blog.css (hunk 20 above)
Comment (Federicofilini) [reply blog.css:201]: Done

File: blog.css (hunk 21 above)
Comment (Federicofilini) [reply blog.css:177]: Done

File: blog.css (hunk 22 above)
Comment (Federicofilini) [reply blog.css:212]: Done

File: blog.css (hunk 23 above)
Comment (Federicofilini) [reply blog.css:217]: Done

File: blog.css (hunk 24 above)
Comment (Federicofilini) [reply blog.css:185]: Done

File: blog.css (hunk 25 above)
Comment (Federicofilini) [reply blog.css:171]: Done

File: blog.css (hunk 26 above)
Comment (Federicofilini) [reply blog.css:222]: Done

File: blog.css (hunk 27 above)
Comment (Federicofilini) [reply blog.css:224]: Done

File: blog.css (hunk 28 above)
Comment (Federicofilini) [reply blog.css:234]: Done

File: blog.html (hunk 29 above)
Comment (Federicofilini) [reply blog.html:195]: Done

File: blog.html (hunk 30 above)
Comment (Federicofilini) [reply blog.html:218]: Done

File: blog.css (hunk 31 above)
Comment (Federicofilini) [reply blog.css:236]: Done

File: blog.css (hunk 32 above)
Comment (Federicofilini) [reply blog.css:239]: Done

File: blog.css (hunk 33 above)
Comment (Federicofilini) [reply blog.css:256]: Done

File: blog.css (hunk 34 above)
Comment (Federicofilini) [reply blog.css:263]: Done

File: blog.css (hunk 36 above)
Comment (Federicofilini) [reply blog.css:306]: Done

File: blog.css (hunk 37 above)
Comment (Federicofilini) [reply blog.css:297]: Done

File: blog.html (hunk 39 above)
Comment (Federicofilini) [reply blog.html:320]: Done

File: blog.html (hunk 40 above)
Comment (Federicofilini) [reply blog.html:321]: Done

File: blog.css (hunk 19 above)
Comment (Federicofilini) [reply blog.css:165]: Done

**Pull request merged**